docs/runtimes/configuring-job-resources.md

```diff
@@ -175,56 +169,56 @@ for each volume to mount to the pod. Multiple volumes can be configured for a si
 
 
 ## Preemption mode: Spot vs. On-demand nodes
-When running ML functions you might want to control whether to run on spot nodes or on-demand nodes. Preemption mode controls 
-whether pods can be scheduled on preemptible (spot) nodes. Preemption mode is supported for all functions. 
-
-Preemption mode uses [Kubernetes Taints and Tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration) to enforce the mode selected.  
 
-### Why preemption mode?
+Spot (preemptible) nodes give you access to spare computing capacity from your cloud environment. 
+With Spot instances, you request capacity from specific availability zones, though it is 
+dependent on spare computing capacity. This is a good choice if you can be flexible about when your applications runs 
+and if your applications can be interrupted. Since spot instances run when there is available capacity, the cost is significantly lower. 
 
-On-demand instances provide full control over the instance lifecycle. You decide when to launch, stop, hibernate, start, 
-reboot, or terminate it. With Spot instances, you request capacity from specific availability zones, though it is
-susceptible to spot capacity availability. This is a good choice if you can be flexible about when your applications run 
-and if your applications can be interrupted. 
-
-Here are some questions to consider when choosing the type of node:
-
-- Is the function mission critical and must be operational at all times?
-- Is the function a stateful function or stateless function?
-- Can the function recover from unexpected failure?
-- Is this a job that should run only when there are available inexpensive resources?
+On-demand instances give you full control over the instance lifecycle. You decide when to launch, stop, hibernate, start, 
+reboot or terminate it. On-demand instances have a fixed (higher) price and are always available. 
 
-```{admonition} Important
-When an MLRun job is running on a spot node and it fails, it won't get back up again. However, if Nuclio goes down due to a spot issue, it 
-is brought up by Kubernetes.
-```
+You can control whether to run your MLRun functions on spot nodes or on-demand nodes. 
+MLRun supports spot nodes for all functions.
 
 Kubernetes has a few methods for configuring which nodes to run on. To get a deeper understanding, see 
 [Pod Priority and Preemption](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption).
 Also, you must understand the configuration of the spot nodes as specified by the cloud provider.
 
-### Stateless and Stateful Applications 
+Preemption mode uses [Kubernetes Taints and Tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration) to enforce the selected mode. 
```

Comment:
It should be noted that the specific taints and tolerations in use differ between the different cloud providers (and in an on-prem deployment may be non-standard). MLRun aims to hide this complexity from the user by creating a standard interface that lets users specify the preemption mode type, and translates it to the underlying k8s constructs per the deployment type.
Users still has the option of manually setting these low-level configurations, given they know the specific configurations that are needed.

Reply:
added